WooCommerce/Classes/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardView.swift

```diff
@@ -8,6 +8,7 @@ import Kingfisher
 final class BlazeCampaignDashboardViewHostingController: SelfSizingHostingController<BlazeCampaignDashboardView> {
     private let viewModel: BlazeCampaignDashboardViewModel
     private let parentNavigationController: UINavigationController?
+    private var blazeNavigationController: UINavigationController?
```

Comment:
Nit: We should use `WooNavigationController` here - a custom navigation controller that hides the title in the back button of the navigation bar.

Reply:
TIL about that class! https://github.com/woocommerce/woocommerce-ios/pull/11625/commits/b4e601e22a6833ad9a953c521a92f0242b539f3c